tutorbackup/templates/backup/build/backup/download_from_s3.py

```diff
@@ -1,22 +0,0 @@
-import os
```

Comment:
Just so I understand this correctly: this one we no longer need because its functionality is now a function in `restore_services.py`, correct?

Reply:
Correct. We download the backup file from S3 [here](https://github.com/hastexo/tutor-contrib-backup/blob/39d68b04397859d8c0cd0e2094eff839bcb89e53/tutorbackup/templates/backup/build/backup/restore_services.py#L110).